src/shared/ask-password-api.c

```diff
@@ -60,7 +60,7 @@ static int lookup_key(const char *keyname, key_serial_t *ret) {
         assert(keyname);
         assert(ret);
 
-        serial = request_key("user", keyname, /* callout_info= */ NULL, /* dest_keyring= */ 0);
+        serial = request_key("user", keyname, /* callout_info= */ NULL, /* destringid= */ 0);
```

Comment:
The man pages request_key(2) says the last argument is `dest_keyring`.
Please update missing_syscall.h, rather than changing here.

Reply:
This seems a bit unfortunate as the syscall in the kernel is defined as [destringid](https://github.com/torvalds/linux/blob/master/include/linux/syscalls.h#L792) so changing `missing_syscall.h` would divert that file from `syscalls.h` in the kernel.

As alternative we could ignore the clang-tidy warning here.